labelbox/client.py

```diff
@@ -120,7 +122,15 @@ def convert_value(value):
         data = json.dumps({'query': query, 'variables': params}).encode('utf-8')
 
         try:
-            response = requests.post(self.endpoint,
+            endpoint = self.endpoint
+
+            if experimental:
+                endpoint = self.endpoint.replace('graphql', '_gql')
+                logger.info(
+                    "Experimental queries/mutations aren't guarenteed to maintain their interface."
```

Comment:
We might want to pass a header in and log it in our analtics so we know how often users are making experimental queries.

Reply:
Also you want  to do `logger.warn` and then explain that their code can break. Also you might want to only do this once per session as to not crowd the users logs

Reply:
This doesn't exist anymore.